R/workhorse.R

```diff
@@ -34,9 +31,8 @@ set_path <- function(x) {
   if (!is.null(x$path)) {
     return(x)
   }
-  ## find the path that matches the given method and resource
-  x$path <- unique(.drive$params$path[.drive$params$method == x$method &
-                                        .drive$params$resource == x$resource])
+  ## find the path that matches the given endpoint
+  x$path <- unique(.drive$params$path[.drive$params$endpoint == x$endpoint])
   return(x)
```

Comment:
Hmm.... I store my API info as a list at this point, with one component per endpoint. Then it has components for `path`, `method` (in the GET, POST, etc.) sense, and `params`, which is a list.

I tend to think this is a good idea, because ... well, that's where I eventually came out 😄 after going through my own tibble phase. But let's not let this block merging this PR. You could open another issue for "refactor storage of API data" or some such, to put this on the radar.